app/src/main/AndroidManifest.xml

```diff
@@ -5,44 +5,45 @@
     >
 
     <application
-        android:name="com.xmartlabs.template.App"
+        android:name=".App"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         >
-
         <activity
-            android:name="com.xmartlabs.template.ui.login.LoginActivity"
-            android:label="@string/login"
+            android:name=".ui.onboarding.signIn.SignInActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/AppTheme.TranslucentStatusBar"
+            ></activity>
+        <activity
+            android:name=".ui.onboarding.signUp.SignUpActivity"
+            android:label="Signup"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme.TranslucentStatusBar"
             />
-
         <activity
-            android:name="com.xmartlabs.template.ui.onboarding.OnboardingActivity"
-            android:label="@string/onboarding"
+            android:name=".ui.login.LoginActivity"
```

Comment:
what's the difference between this activity and `".ui.onboarding.signIn.SignInActivity"` declared above?

Reply:
also, why remove the onboarding activity?

Reply:
It is LoginActivity belongs to base project